backend/src/helpers/cars/api_autoria.helper.ts

```diff
@@ -0,0 +1,23 @@
+import { ENV } from '@common/enums/app/env.enum';
+import { AutoriaRequestParams } from '@common/types/types';
+import axios from 'axios';
+
+const URL = 'https://developers.ria.com/auto/search';
```

Comment:
Maybe better move it to env? @oleksandr-danylchenko

Reply:
I think it's okay to leave it in the source code. It won't change depending on ENV and I have high hopes that the autoria api URL won't change in a next month